study-builder/studies/brugada/snippets/picklist-0-to-4.conf

```diff
@@ -0,0 +1,64 @@
+{
+  "picklistOptions": [
```

Comment:
Why do we need to have two separate PL options (0-4 and 1-6) ?

Reply:
Can we please have more meaningful name rather than numerics that doesn't tell what these options are about

Reply:
It is named the way it is: It's just a scales for a lot of questions. What name could be more clear?
<img width="1471" alt="image" src="https://user-images.githubusercontent.com/71830706/135411626-779760c3-ffe4-42c2-acd5-2d57ba88c089.png">